ykrt/src/compile/jitc_llvm/deopt.rs

```diff
@@ -418,7 +418,7 @@ unsafe extern "C" fn __ykrt_deopt(
                 aotvalslen: aotvals.length,
                 guardid,
             };
-            ctr.mt().side_trace(sti, ctrn);
+            //ctr.mt().side_trace(sti, ctrn);
```

Comment:
Not sure if this change is intentional: should this now be a `panic` perhaps?

Reply:
I thought we will just delete the llvm backend soon, so I didn't think it mattered too much. But can fix you want met to.

Reply:
I get very nervous about code that silently fails, just in case we don't delete it as soon / in the manner we expect. So a `panic` (or similar) would be good!

Reply:
[b73a7b3](https://github.com/ykjit/yk/pull/1287/commits/b73a7b3a51412ce8d889b22976f8101e4c4c0ecc)